### DISCUSSION TITLE
应用代理的问提

### DISCUSSION THREAD

**Discussion by xlwmin** [2dust/v2rayNG#3123] [General]
在提出问题前请先自行排除服务器端问题，同时也请通过搜索确认是否有人提出过相同问题。

### 预期行为
其他应用程序，能通过我打包的v2rayNg实现代理。

### 实际行为
我打包的V2rayNG，测试连接的时候是能成功连接的，但是其他应用却无法通过vpn连接网络。改变代理模式也不行。

### 日志信息
<details>

05-15 04:25:13.476   649   649 I GoLog   : loading core config
05-15 04:25:13.477   649   715 I GoLog   : Preparing Domain: xxx.xxx
05-15 04:25:13.481   649   649 I GoLog   : new core
05-15 04:25:13.482   649   649 I GoLog   : start core
05-15 04:25:13.484   649   727 I GoLog   : [Warning] core: Xray 1.8.10 started
05-15 04:25:13.657   649   715 I GoLog   : Prepare Result:
05-15 04:25:13.657   649   715 I GoLog   :  Domain: xxx.xxx
05-15 04:25:13.657   649   715 I GoLog   :  Port: 8880
05-15 04:25:13.657   649   715 I GoLog   :  IPs: [114.18.12.229 114.18.13.229]
05-15 04:25:21.330   649   718 I GoLog   : Using Prepared: 104.18.12.229  这里是测试连接 测试连接能成功，但其他应用无法通过代理
05-15 04:25:26.659   649  1142 I GoLog   : Using Prepared: 104.18.12.229 这里是测试连接
05-15 04:25:29.234   649  1142 I GoLog   : Using Prepared: 104.18.12.229 这里是测试连接
05-15 04:26:40.040   649  1142 I GoLog   : Using Prepared: 104.18.12.229 这里是测试连接
</details>

![log](https://github.com/2dust/v2rayNG/assets/5059521/1dd45115-c432-4dae-863e-625915b68d59)
![sign](https://github.com/2dust/v2rayNG/assets/5059521/d8d2d430-b31d-4e3b-a9e4-72b583d6a5aa)

### 环境信息
windows 10
jdk 21
ndk 26
go 1.22.3
使用的是[AndroidLibXrayLite](https://github.com/xlwmin/AndroidLibXrayLite) 这个编译的libv2ray.aar包
### 额外信息（可选）
还有就是，这个libv2ray.aar签名是错误的，是不是这个原因影响代理问题的呢？

**Comment by 13237679843** [comment]
请问该问题是否解决，现在好像arr都编译不了
